docs/docs/policies.mdx

```diff
@@ -346,8 +346,56 @@ policies:
 * `enable_fallback_prediction` (default: `true`): Please see the
    [fallback documentation](fallback-handoff.mdx#handling-low-action-confidence) for
    further information.
-* `check_for_contradictions` (default: `true`): After training, the RulePolicy will
-   perform a check to make sure that there are no rules that contradict with each other, or with any stories.
+* `check_for_contradictions` (default: `true`):
+   Before training, the RulePolicy will perform a check to make sure that
+   actions always set slots and active loops that they set in other rules.
+   The following snippet is an example for an incomplete rule:
+
+   ```yaml-rasa
+   rules:
+   - rule: complete rule
+     steps:
+     - intent: do_something
+     - action: action_that_sets_some_slot
+     - slot_was_set:
+       - some_slot: "some value"
```

Comment:
```suggestion
       - venues: [{"name": "Big Arena", "reviews": 4.5}]
```

Reply:
this is a weird format. Can a slot be a dictionary?

Reply:
Yeah, it's a list of dicts. I'm pulling it from the concertbot example.